test/integration_test.tsx

```diff
@@ -469,8 +527,98 @@ describe('integration tests', () => {
           <Component />
         </div>
       );
-      const output = wrapper.debug().replace(/\s+/g, '');
-      assert.equal(output, '<div><Component/></div>');
+      const output = normalizeDebugMessage(wrapper.debug());
+      assert.equal(output, '<div><Component /></div>');
+    });
+
+    it('renders wrapped component with wrapper and props', () => {
+      function Component() {
+        return <span>test</span>;
+      }
+      const wrapper = shallow(<Component />, {
+        wrappingComponent: WrappingComponent,
+        wrappingComponentProps: { foo: 'bar' },
+      });
+
+      const output = debugWrappedShallowComponent(wrapper);
+      assert.equal(
+        output,
+        '<div foo="bar"><RootFinder><Component /></RootFinder></div>'
+      );
+    });
+
+    it('passes wrappingComponentProps to wrappingComponent', () => {
```

Comment:
Note: A bit late because this PR is already merged but: I just realized that this test is basically a duplicate of the previous test. I used to run different testing logic in here but after some refactor I hadn't realized that it has become a redundant test. I'd suggest removing it in a future update.

Reply:
Would you mind submitting a small follow-up PR?

Reply:
Done: https://github.com/preactjs/enzyme-adapter-preact-pure/pull/158